src/lib.rs

```diff
@@ -334,6 +336,57 @@ impl Builder {
     }
 }
 
+impl Debug for Builder {
```

Comment:
I suggest that we add Debug implementations as needed. The value of Builder is set by user, and I think it is rarely needed to print it.

Reply:
OK, I can remove this.